.github/workflows/android-build.yml

```diff
@@ -20,6 +20,15 @@ jobs:
     steps:
       - name: checkout
         uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: '11'
+      - name: Install NDK
```

Comment:
How does this affect the execution time of this job? Is it possible to cache this environment in order to optimize it?

Reply:
This adds 1m 10s to test duration. To be clear - this is not necessary, but I just wanted to provide a deterministic environment.

I found a workaround - https://stackoverflow.com/a/62173092/7276742

Reply:
Update according to documentation nttld/setup-ndk@v1 action caching NDK by default:
https://github.com/nttld/setup-ndk - "This action sets up an Android NDK environment by downloading and caching a version of the NDK and adding it to the PATH"